code/modules/media/broadcast/transmitter.dm

```diff
@@ -34,6 +34,9 @@ var/global/media_transmitters=list()
 			if(R.media_crypto == media_crypto)
 				R.receive_broadcast(media_url,media_start_time)
 				//testing("[src]: Sending music to [R]")
+		for(var/mob/living/carbon/not_human/gondola/G in media_receivers[freq])
+			if(G.media_crypto == media_crypto)
+				G.receive_broadcast(media_url,media_start_time)
```

Comment:
iirc media receivers do not play well with crossing room boundaries (e.g.: boombox) and simply won't update their playing area unless turned off and on